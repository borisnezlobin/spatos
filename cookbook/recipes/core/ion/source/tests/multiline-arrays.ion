let x = [
    a # a comment
    b # another comment
    ab#cd #with a comment
]

echo @x

let y = [
    a
    # a comment
    #another comment
]

echo @y
