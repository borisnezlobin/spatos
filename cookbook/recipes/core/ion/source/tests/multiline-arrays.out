a b ab#cd
a
